org.emoflon.ibex.tgg.ide/src/org/emoflon/ibex/tgg/ide/admin/IbexTGGBuilder.java

```diff
@@ -302,6 +302,20 @@ public static Resource loadModelInProject(IFile file, ResourceSet rs) throws IOE
 		resource.load(null);
 		return resource;
 	}
+	
+	/**
+	 * This method returns a resource that has already been loaded into the resourceset
+	 * @param file
+	 * @param rs
+	 * @return
+	 * @throws IOException
+	 */
+	public static Resource getModelInProject(IFile file, ResourceSet rs) throws IOException {
+		URI uri = URI.createPlatformResourceURI(file.getProject().getName() + "/" + file.getProjectRelativePath().toString(), true);
+		Resource resource = rs.getResource(uri, true);
+		resource.load(null);
```

Comment:
This line doesn't make much sense.  Using ```getResource(uri, true)``` already loads the resource on demand.
So why both reloading?  I think this can (probably should) just be removed